internal/log/context.go

```diff
@@ -25,22 +25,60 @@ func ContextWithAttrs(ctx context.Context, attr ...slog.Attr) context.Context {
 	return context.WithValue(ctx, contextAttrSliceKey, attrSlice)
 }
 
+func ClearContextAttrs(ctx context.Context) context.Context {
+	attrSlice := attrSliceFromContext(ctx)
```

Comment:
I know the change was in a previous PR but should we use a more unique key than `struct{}{}` for `contextAttrSliceKey`? `struct{}{}` seems more likely to have a potential clash with third party code.